CHANGELOG.md

```diff
@@ -4,6 +4,7 @@ Bob versions changelog
 
 ## [Unreleased]
 #### Added
+- Lookup existence in aliens, separate local and remote lookup (#576, #585)
```

Comment:
Please, split this line into 2: first for issue #576 and second for #585

Reply:
Done.